scripts/resource/RoomData.gd

```diff
@@ -35,16 +37,16 @@ func calculate_tile_count(vector1: Vector2i, vector2: Vector2i) -> int:
 
 func calculate_power_consumption() -> int:
 	var tile_count = calculate_tile_count(top_left, bottom_right)
-	return tile_count * room_type.power_consumption
+	return tile_count * type.power_consumption
 	
 func generate_hotspots() -> void:
 	hot_spots.clear()
-	var capacity = room_type.capacity
-	var tile_count = calculate_tile_count(top_left, bottom_right)
+	var capacity = type.capacity
+	var _tile_count = calculate_tile_count(top_left, bottom_right)
 	var available_tiles = []
 	
 	var min_x = min(top_left.x, bottom_right.x)
-	var max_x = max(top_left.x, bottom_right.x) + 1
+	var _max_x = max(top_left.x, bottom_right.x) + 1
```

Comment:
Why only for this one?

Reply:
```suggestion
        var _tile_count = calculate_tile_count(top_left, bottom_right)
        ...
	var _max_x = max(top_left.x, bottom_right.x) + 1
```
not just for that one, there were two variable that were not being used but i wasn't sure if they should be removed or if you planned on using them for something later on so i left it. the underscore marks it as an unused variable and stops warnings being shown.

Reply:
@conalli Sorry. I didn't fix this line after trying to figure out why the range wasn't working:

`for x in range(min_x, bottom_right.x + 1):`

It should be max_x here.